rush/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -15,6 +15,16 @@ import Utilities from '../utilities/Utilities';
 import { RushConstants } from '../RushConstants';
 
 /**
+ * A list of known config filenames that are expected to appear in the "./common/rush/config" folder.
```

Comment:
Ideally the GCB configs should behave the same way.  It is a very poor user experience for config files to just be silently ignored, merely because someone misspelled the name, or because a rig happens to be missing a certain task.

@qz2017

---
In reply to: [115864980](https://github.com/Microsoft/web-build-tools/pull/176#discussion_r115864980) [](ancestors = 115864980)